student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+# class student inherite person class
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Since you are calling the `super` method here and the `parent_permission` method is a **_keyword_** argument, you ought to reference the `_parent_permission` parameter in the parent class and pass it a value of the `parent_permission` parameter that is being created for the `Student` class. This will map the variables properly and the `super` method will use the conditions set in the `initialize` method of the `Person` class to be the same condition as those passed as parameters in the `super` method. Kindly implement this change so that you don't get these errors 💻 🖱️:

![image](https://github.com/DianaBeki/school-library/assets/77106962/4844c022-f46c-4775-a358-9773a21e81fa)

![image](https://github.com/DianaBeki/school-library/assets/77106962/92f525cd-005e-4ad1-bf72-7addc0537f66)